Sports/live_tennis.1m.py

```diff
@@ -1,111 +1,220 @@
 #!/usr/bin/env python3
 
 # <xbar.title>Live Tennis Scores</xbar.title>
-# <xbar.version>v1.1</xbar.version>
+# <xbar.version>v2.0</xbar.version>
 # <xbar.author>Anup Sam Abraham</xbar.author>
 # <xbar.author.github>anupsabraham</xbar.author.github>
 # <xbar.desc>Show live scores for tennis matches using ATP World Tour api</xbar.desc>
-# <xbar.image>http://i.imgur.com/5kOPKVv.png</xbar.image>
-# <xbar.dependencies></xbar.dependencies>
+# <xbar.image>https://i.postimg.cc/BQq9CSgv/SCR-20231231-mlck.png</xbar.image>
+# <xbar.dependencies>python</xbar.dependencies>
 # <xbar.abouturl></xbar.abouturl>
 
+import sys
+import getopt
+import os
 import json
-from urllib.request import Request, urlopen
+import http.client
+import gzip
 
-atpworldtour_base_url = "http://www.atpworldtour.com"
-inital_scores_url = atpworldtour_base_url + "/en/-/ajax/Scores/GetInitialScores"
+FAVORITE_PLAYERS_FILE = "/var/tmp/bitbar_tennis_favourites.data"
```

Comment:
This might be a good use case for [xbar variables](https://xbarapp.com/docs/2021/03/14/variables-in-xbar.html). Let the user set where they want the file saved.

Reply:
Should it be completely configurable by the user or are you suggesting to use something like a dropdown options? 
The code should also verify if the path exists, right?